src/teaching-element/Assessment/MultipleChoice.vue

```diff
@@ -3,58 +3,76 @@
     <span class="form-label">Solution</span>
     <ul class="answers">
       <li
-        v-for="(answer, index) in answers"
-        :key="index"
-        :class="{ selected: isSelected(index) }">
+        v-for="(answer, index) in mappedAnswers"
+        :key="answer.id"
+        :class="getAnswerClass(index)">
         <input
           v-model="userAnswer"
-          :value="index"
+          :id="answer.id"
           :disabled="disabled"
+          :value="index"
           @change="update"
           class="answers-checkbox"
           type="checkbox">
-        <span class="order">{{ transform(index) }}.</span>
-        <span>{{ answer }}</span>
+        <label :for="answer.id">{{ transform(index) }}. {{ answer.answer }}</label>
```

Comment:
`answer` and `id` should be destructured from the `answer` object.

Reply:
👍